game.py

```diff
@@ -13,7 +13,7 @@ def play_game(self):
         while True:
             self.board.disp_board()
             choice = self.players[self.turn].get_choice(self.board)
-            self.board.add_piece(choice)
+            self.board.add_piece(choice)  # add_piece has 2 parameters
```

Comment:
Your program crashes when I play a round because add_choice needs the column AND the piece